app/dashboards/article_dashboard.rb

```diff
@@ -0,0 +1,69 @@
+require "administrate/base_dashboard"
+
+class ArticleDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    id: Field::Number,
```

Comment:
I don't think we need to add/show the `id` 
In a way, I find that the `id` is for the db and the routes, but it's not for the user

Reply:
I have published some new changes to remove the `id`.